Dockerfile

```diff
@@ -32,6 +32,7 @@ COPY . $CODE_DIR
 # set this to 'production' in production deploys
 ENV ENVIRONMENT dev
```

Comment:
Is this redundant with the new IS_PRODUCTION environment variable?

Reply:
mmm yeah this isn't really used anywhere, might as well remove it.